redirects.csv

```diff
@@ -1,5 +1,3719 @@
 http://crossroads.net/*,https://www.crossroads.net/:splat,301!,master
 http://${env:CRDS_APP_DOMAIN}/*,https://${env:CRDS_APP_DOMAIN}/:splat,301!
+/real,https://int.crossroads.net/media/series/real-encounters-with-god,302
```

Comment:
3700 new redirects? This is a mistake, right?

Reply:
100% a mistake

Reply:
@tcmacdonald Fixed